### ISSUE TITLE
Projector Broken in Classroom 1

### ISSUE THREAD

**Issue by mattscilipoti** [ga-dc/campus#24]
This is a known issue (we have projector on cart).  Adding to ensure it makes it into the regular flow.

**Comment by jshawl** [comment]
https://generalassembly.slack.com/archives/dc-instructors/p1448930062000005

**Comment by mattscilipoti** [comment]
Tested w/FL today.  Works with Matt's laptop, though it is still dim.  The temporary projector is brighter, but not as sharp.  Identified that installed projector needs bulb.  Made decision to continue with temporary projector until that is corrected.
Closing this ticket as duplicate of #14.

**Comment by timfoley** [comment]
I've heard from my instructors that the overhead projector in CR 1 doesn't work. Right now they're using a podium projector. Any idea on the status of that projector? Did the new bulb help?

**Comment by Campus-Commander** [comment]
@amychoi @timfoley Hey Tim! Amy met with Anthony from IT earlier this week when he made a site visit. She can connect with you and provide deeper insight.

**Comment by amychoi** [comment]
We are getting a new bulb. The other issue we found out when the AV people
came was that the HDMI cord from the projector through the wall to the wall
plate is backwards, hence why the picture is always crappy. They will be
replacing the HDMI cord and replacing the bulb as soon as we receive it.
Hopefully done by the first week of February.

*Amy Choi *| Experience Coordinator
General Assembly http://generalassemb.ly/ | 1133 15th St. NW |
Washington, D.C.
amy.choi@ga.co | 202-823-3874

**Comment by shanazcw** [comment]
Hey @amychoi Similar update here. :)

**Comment by bmartinowich** [comment]
@ShanazFC IT is coming on Weds to fix the cord. Projector is being RMA'd to be replaced. Will keep open for the time being, unless the cord resolves the problem.

**Comment by bmartinowich** [comment]
@amychoi Was IT able to fix the cord/projector last week? Seemed like we had issues today

**Comment by amychoi** [comment]
Yes - they replaced the HDMI cord that was faulty in the wall. We will also
just be replacing that projector when we receive a new one from Austin's
campus. Still waiting on the ETA for that and to reschedule MyerConnex.

*Amy Choi *| Experience Coordinator
General Assembly http://generalassemb.ly/ | 1133 15th St. NW |
Washington, D.C.
amy.choi@ga.co | 202-823-3874

**Comment by bmartinowich** [comment]
@amychoi I believe that the projector has been replaced. Can you confirm that this has been resolved? cc: @mattscilipoti

**Comment by amychoi** [comment]
Yes - it's been replaced.

*Amy Choi *| Experience Coordinator
General Assembly http://generalassemb.ly/ | 1133 15th St. NW |
Washington, D.C.
amy.choi@ga.co | 202-823-3874

On Wed, Feb 17, 2016 at 4:51 PM, Brian Martinowich <notifications@github.com
